src/redux/banner/methods.ts

```diff
@@ -10,7 +10,7 @@ import { BannerActionType, BannerState, SetBannerAction } from './types'
 export const setBanner = (state: BannerState): void => {
   const action: SetBannerAction = {
     type: BannerActionType.SET_BANNER,
-    state
+    state: state
```

Comment:
This shouldn't be needed and is inconsistent with other redux set methods.

Reply:
Actually it would be better if we add this to the other actions. So you can't mix up actions with wrong interfaces

Reply:
Hm? This is just the ES6 short syntax?

Reply:
That's typescript. If you define a type in an superior interface and a specific value type in children interfaces then typescript can type interfer the other attributes of the interfaces.

Reply:
Oh you mean THIS! sry I though you meant something else. Ignore my comment then.